src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery firstTime = new Lottery();
```

Comment:
What is firstTime ? lottery

Reply:
firstTime is an object of the Lottery class. I don't understand, where is the mistake here?

Reply:
Inappropriate name for object